src/main/resources/assets/tconstruct/book/materials_and_you/index.json

```diff
@@ -13,25 +13,44 @@
     "data": "sections/tools.json",
     "icon": {
       "item": {
-        "id": "minecraft:iron_pickaxe",
-        "damage": 0
+        "id": "tconstruct:pickaxe",
+        "nbt": {
+          "tic_broken": false,
+          "tic_display_tool": true,
+          "Damage": 0,
+          "tic_materials": [
+            "tconstruct:ui_render_head",
+            "tconstruct:ui_render_handle",
+            "tconstruct:ui_render_extra",
+            "tconstruct:ui_render_large"
+          ]
+        }
       }
     }
   },
   {
-    "name": "tier_one_materials",
+    "name": "modifiers",
+    "data": "sections/modifiers.json",
     "icon": {
       "item": {
-        "id": "minecraft:stone"
+        "id": "minecraft:redstone"
       }
     }
   },
   {
-    "name": "modifiers",
-    "data": "sections/modifiers.json",
+    "name": "further_reading",
+    "data": "sections/furtherreading.json",
     "icon": {
       "item": {
-        "id": "minecraft:redstone"
+        "id": "tconstruct:puny_smelting"
+      }
+    }
+  },
+  {
+    "name": "tier_one_materials",
```

Comment:
Lets put materials before modifiers. You cannot modify a tool if you have not built it yet.